.aws/src/main.ts

```diff
@@ -65,13 +65,13 @@ class SnowplowSharedConsumerStack extends TerraformStack {
       tags: config.tags,
     });
 
-    const userEventTopicArn = `arn:aws:sns:${region.name}:${caller.accountId}:${config.eventBridge.prefix}-${config.environment}-${config.eventBridge.userTopic}`;
-    this.subscribeSqsToSnsTopic(
-      sqsEventLambda,
-      snsTopicDlq,
-      userEventTopicArn,
-      config.eventBridge.userTopic
-    );
+    // const userEventTopicArn = `arn:aws:sns:${region.name}:${caller.accountId}:${config.eventBridge.prefix}-${config.environment}-${config.eventBridge.userTopic}`;
```

Comment:
commented out instead of removing as the handlers are in the repo and we might need it soon